migrator/liquibase-postgresql-example.properties

```diff
@@ -0,0 +1,5 @@
+url=jdbc:postgresql://localhost:5432/ups_dev100li
+driver=org.postgresql.Driver
```

Comment:
we use a different class here:
https://github.com/aerogear/aerogear-unifiedpush-server/blob/master/databases/postgresql-database-config.cli#L6